content/compose/release-notes.md

```diff
@@ -11,6 +11,28 @@ aliases:
 
 For more detailed information, see the [release notes in the Compose repo](https://github.com/docker/compose/releases/).
 
+## 2.22.0
+{{< release-date date="2023-09-21" >}}
+
+> **Note**
+>
+> `watch` command is now GA and can be directly used from the root command `docker compose watch`
```

Comment:
@glours Thanks for the PR. Could you add a link to the Compose watch docs/CLI reference here?

Reply:
Done!